apps/docusaurus/docs/create-aptos-dapp/faq.md

```diff
@@ -16,3 +16,49 @@ the move module is trying to use a module that is not live yet
 ## I tried to publish my dapp to a live server but getting `404 error`
 
 might need to update the root route, if you deployed your site to `user-name.github.io/my-repo` then root route should be updated to `my-repo`
+
+## What is Tailwind CSS?
+
+Tailwind is a utility-first CSS framework that scans your components for class names and generates a static CSS file containing the corresponding styles at build-time.
+
+This framework makes it easy to quickly author styles that are co-located with your component markup without any incurring any runtime performance costs. It also helps you to maintain a consistent theme throughout your app that is responsive to light and dark mode.
+
+To learn more about Tailwind CSS, please refer to their official [documentation](https://tailwindcss.com/docs/utility-first).
+
+## What is `shadcn/ui`?
+
+Shadcn is a collection of accessible components that you can copy and paste into your app through their CLI tool. Since the source files live in your app's codebase, you can customize them as much as you need to.
+
+These components are built on top of [Radix UI Primitives](https://www.radix-ui.com/primitives) and are styled with [Tailwind CSS](https://tailwindcss.com/). To learn more about `shadcn/ui`, please refer to their official [documentation](https://ui.shadcn.com/docs).
```

Comment:
I think there's value in having a brief summary/prelude for those skimming our docs rather than immediately linking them somewhere else. This context also gives a bit of insight to why we chose these technologies for the UI.